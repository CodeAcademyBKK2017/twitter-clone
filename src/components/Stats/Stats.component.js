import React, {Component} from 'react';
import classNames from 'classnames';
import statsStyle from './Stats.style.css';

class Stats extends Component {
  render () {
    return (
      <div className={classNames(statsStyle.navContainer)}>
        <div className={classNames(statsStyle.nav, statsStyle.active)}>
          <div className={classNames(statsStyle.nav__name)}>tweets</div>
          <div className={classNames(statsStyle.nav__value)}>200</div>
        </div>
        <div className={classNames(statsStyle.nav)}>
          <div className={classNames(statsStyle.nav__name)}>photos/videos</div>
          <div className={classNames(statsStyle.nav__value)}>200</div>
        </div>
        <div className={classNames(statsStyle.nav)}>
          <div className={classNames(statsStyle.nav__name)}>following</div>
          <div className={classNames(statsStyle.nav__value)}>200</div>
        </div>
        <div className={classNames(statsStyle.nav)}>
          <div className={classNames(statsStyle.nav__name)}>followers</div>
          <div className={classNames(statsStyle.nav__value)}>1m</div>
        </div>
      </div>
    );
  }
}

export default Stats;