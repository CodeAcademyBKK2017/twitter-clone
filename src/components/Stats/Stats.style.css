.navContainer {
    display: flex;
}
.nav {
    height: 50px;
    flex: 1;
    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;
    border-bottom: 4px solid transparent;
}
.nav.active {
    border-bottom: 4px solid #81a9cb;
}
.nav:hover {
    border-bottom: 4px solid #81a9cb;
}
.nav:active {
    border-bottom: 4px solid red;
}
.nav__name {
    text-transform: uppercase;
    padding: 1px 0;
    /* font-size: 0.9vw; */
    font-size: 0.7em;
    color: lightgray;
}
.nav__value {
    text-transform: uppercase;
    font-weight: bold;
    /* font-size: 1vw; */
    font-size: 0.8em;
    color: darkgray;
}
@media only screen and (max-width : 768px) {
    .navContainer {
        flex-direction: column;
    }
    .nav {
    }
}
