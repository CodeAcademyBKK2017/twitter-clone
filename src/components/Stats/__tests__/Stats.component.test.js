import React from 'react';
import renderer from 'react-test-renderer';
import Stats from '../Stats.component';

it('Cell: Snapshot test', () => {
  const snapshot = renderer.create(<Stats />).toJSON();
  expect(snapshot).toMatchSnapshot();
});