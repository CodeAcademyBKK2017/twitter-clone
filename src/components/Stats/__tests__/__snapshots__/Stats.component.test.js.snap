// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell: Snapshot test 1`] = `
<div
  className=""
>
  <div
    className=""
  >
    <div
      className=""
    >
      tweets
    </div>
    <div
      className=""
    >
      200
    </div>
  </div>
  <div
    className=""
  >
    <div
      className=""
    >
      photos/videos
    </div>
    <div
      className=""
    >
      200
    </div>
  </div>
  <div
    className=""
  >
    <div
      className=""
    >
      following
    </div>
    <div
      className=""
    >
      200
    </div>
  </div>
  <div
    className=""
  >
    <div
      className=""
    >
      followers
    </div>
    <div
      className=""
    >
      1m
    </div>
  </div>
</div>
`;
